### ISSUE TITLE
Validation with a schema using "format" causes constraints on other elements to not apply in validation.

### ISSUE THREAD

**Issue by MohammadHossein-Namjoo** [danielaparker/jsoncons#524]
**When validating with a JSON Schema that uses "format" in one of its properties, other properties are validated (even with wrong values)**

Example schema: [exampleSchema.json](https://github.com/user-attachments/files/15808006/exampleSchema.json)

Example JSON to validate:

> {
    "ipv4": "192.168.1.123",
    "someNumber": 256
}

**When validating the given JSON against the given schema**
*Expectation:*
The validation should fail because 256 is bigger than the maximum specified in the schema.

*Actual Behavior:*
The validation passes.

        ojson schema = ojson::parse(schemaStringStream.str());

        jsonschema::json_schema<ojson> compiledSchema =
            jsonschema::make_json_schema(schema,
                                         Resolver,
                                         jsonschema::evaluation_options()
                                             .require_format_validation(true));

```
    const std::string INVALID_DATA = R"(
{
    "ipv4": "192.168.1.123",
    "someNumber": 256
}
    )";
```

        compiledSchema.is_valid(ojson::parse(INVALID_DATA)));

**NOTE**
If the schema is changed to use a "pattern" instead of the "format", this issue does not happen.

**What compiler, architecture, and operating system?**

- Compiler: MSVC
- Architecture: x64
- Operating system: Windows 10

**What jsoncons library version?**
master - f393930fb58a884e317fa50a19faa441a3a998b5

**Comment by danielaparker** [comment]
I believe that's the same as #520, fixed in latest release 0.176.0. (https://github.com/danielaparker/jsoncons/commit/f393930fb58a884e317fa50a19faa441a3a998b5 predates the fix)

**Comment by danielaparker** [comment]
I'm closing this as it appears to be a duplicate of an issue already fixed.
